library/src/enabled/AndroidManifest.xml

```diff
@@ -1,7 +1,5 @@
 <manifest
-		xmlns:android="http://schemas.android.com/apk/res/android"
-		package="com.telefonica.androidlogger"
```

Comment:
don't we need to add the `namespace` in library/build.gradle just like in app module?

Reply:
You can find it here: https://github.com/Telefonica/android-logger/pull/13/files#diff-1cfc99164a9e87fb25dc437e1caa400c8144dcec3cf57c8766f7498e5957028dR33